units/stats_host_idle.go

```diff
@@ -182,6 +182,7 @@ func (j *collectHostIdleDataJob) getHostStatsMessage(cost float64, idleTime time
 
 	if j.host.Status == evergreen.HostTerminated {
 		msg["task_count"] = j.host.TaskCount
+		msg["total_idle_secs"] = j.host.TotalIdleTime.Seconds()
```

Comment:
Is this to check there's not a problem from splunk graphs?

Reply:
This is an unrelated change that Sam requested to make the stats reporting more meaningful